src/Modal/styles.css

```
@@ -0,0 +1,35 @@
+.modal {
+  position: fixed;
+  top: 50%;
+  left: 50%;
```

```suggestion
  left: -50%;
```

Super cool bounce animation that looks like in-game part 1 😎